www/docs/en/dev/config_ref/index.md

```diff
@@ -45,207 +45,356 @@ passively copied into various `platforms/` subdirectories, for example:
+   
+   Examples:
+
+    <preference name="DisallowOverscroll" value="true"/>
+    <preference name="Fullscreen" value="true" />
+    <preference name="BackgroundColor" value="0xff0000ff"/>
+    <preference name="HideKeyboardFormAccessoryBar" value="true"/>
+    <preference name="Orientation" value="landscape" />
+    
+    <!-- iOS only preferences -->
+    <preference name="EnableViewportScale" value="true"/>
+    <preference name="MediaPlaybackAllowsAirPlay" value="false"/>
+    <preference name="MediaPlaybackRequiresUserAction" value="true"/>
+    <preference name="AllowInlineMediaPlayback" value="true"/>
+    <preference name="BackupWebStorage" value="local"/>
+    <preference name="TopActivityIndicator" value="white"/>
+    <preference name="SuppressesIncrementalRendering" value="true"/>
+    <preference name="GapBetweenPages" value="0"/>
+    <preference name="PageLength" value="0"/>
+    <preference name="PaginationBreakingMode" value="page"/>
+    <preference name="PaginationMode" value="unpaginated"/>
+    <preference name="UIWebViewDecelerationSpeed" value="fast" />
+    <preference name="ErrorUrl" value="myErrorPage.html"/>
+    <preference name="OverrideUserAgent" value="Mozilla/5.0 My Browser" />
+    <preference name="AppendUserAgent" value="My Browser" />
+    <preference name="target-device" value="universal" />
+    <preference name="deployment-target" value="7.0" />
+    <preference name="CordovaWebViewEngine" value="CDVUIWebViewEngine" />
+    <preference name="SuppressesLongPressGesture" value="true" />
+    <preference name="Suppresses3DTouchGesture" value="true" />
+    
+    <!-- Android only preferences -->
+    <preference name="KeepRunning" value="false"/>
+    <preference name="LoadUrlTimeoutValue" value="10000"/>
+    <preference name="SplashScreen" value="mySplash"/>
+    <preference name="InAppBrowserStorageEnabled" value="true"/>
+    <preference name="LoadingDialog" value="My Title,My Message"/>
+    <preference name="ErrorUrl" value="myErrorPage.html"/>
+    <preference name="ShowTitle" value="true"/>
+    <preference name="LogLevel" value="VERBOSE"/>
+    <preference name="AndroidLaunchMode" value="singleTop"/>
+    <preference name="DefaultVolumeStream" value="call" />
+    <preference name="OverrideUserAgent" value="Mozilla/5.0 My Browser" />
+    <preference name="AppendUserAgent" value="My Browser" />
+    
+    <!-- Windows only preferences -->
+    <preference name="windows-phone-target-version" value="8.1" />
+    <preference name="windows-target-version" value="8.1" />
+    <preference name="Windows.Universal" value="10.0.10240.0" />
+
+    <!-- BlackBerry only preferences -->
+    <preference name="ChildBrowser" value="disable"/>
+    <preference name="PopupBlocker" value="enable"/>
+    <preference name="WebSecurity" value="disable"/>
+    
+## feature
```

Comment:
So this tag is only used by plugin authors, right? They insert it into the platform-level `config.xml` using `plugin.xml`'s `<config-file />` tag. Do Cordova app developers ever have to manually add this to a config.xml file? If not, we should make that very clear; this paragraph is a little confusing.